dataRetriever/blockchain/finalCoordinates.go

```diff
@@ -0,0 +1,7 @@
+package blockchain
+
+type finalCoordinates struct {
```

Comment:
this is a simple dto, can be placed inside blockchain.go

Reply:
Moved and renamed to `blockInfo`.